src/trigger-method.js

```diff
@@ -33,11 +33,7 @@ Marionette._triggerMethod = (function() {
 
     // trigger the event, if a trigger method exists
     if (_.isFunction(context.trigger)) {
-      if (noEventArg + args.length > 1) {
-        context.trigger.apply(context, noEventArg ? args : [event].concat(_.drop(args, 0)));
-      } else {
-        context.trigger(event);
-      }
+      context.trigger.apply(context, noEventArg ? args : [event].concat(_.drop(args, 0)));
```

Comment:
Nope, `drop(0)` is the only consistent way to do a fast slice on an arguments object in both lodash>=2 and underscore^1.4.2

@jridgewell its a faster way of doing `toArray` in underscore currently